Geometry/CMSCommonData/data/dd4hep/cmsExtendedGeometry2021.xml

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0"?>
 <DDDefinition>
   <open_geometry/>
-  <close_geometry/>
+  <!--close_geometry/-->
```

Comment:
`Geometry/CMSCommonData/data/dd4hep/cmsExtendedGeometry2021.xml` is automatically generated, is it not? This change will be lost after the next change to 2021 geometry. Is that intended?

Reply:
That's a good point - unit tests may fail now.

Reply:
done in https://github.com/cms-sw/cmssw/pull/31915

Reply:
BTW, these tests are not linked to the Geometry subsystem. The unit tests did not fail during integration of this PR, but later, when an IB was done...

Reply:
Indeed; right now, we don't have a good way to make the test from `Configuration/Geometry` depend on changes in xml files in `Geometry/CMSCommonData/data/dd4hep`.